### PR TITLE
asm-1114 - zero trust web access helm chart

### PR DESCRIPTION
**Pull request by liorhasson**

### CODE REVIEW

File: akeyless-zero-trust-web-access/templates/dispatcher-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        - name: akeyless-docker-hub
+      containers:
+        - name: {{ .Values.dispatcher.containerName }}
+          image: "{{ .Values.dispatcher.image.repository }}:{{ .Values.dispatcher.image.tag }}"
+          imagePullPolicy: {{ .Values.dispatcher.image.pullPolicy }}
+{{/*          imagePullPolicy: Never*/}}
+          ports:
+            - containerPort: 9000
+              name: web
+            - containerPort: 12411
```
Comment (OriMankali) [inline akeyless-zero-trust-web-access/templates/dispatcher-deployment.yaml:53]: Can you explain how is this port not exposed to the world?

File: akeyless-zero-trust-web-access/templates/dispatcher-service.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+{{- toYaml .Values.dispatcher.service.annotations | nindent 4 }}
+spec:
+  type: LoadBalancer
+  ports:
+    - name: http
+      port: {{ .Values.dispatcher.service.port }}
+      targetPort: web
+      protocol: TCP
+    - name: akl
+      port: 12411
```
Comment (OriMankali) [inline akeyless-zero-trust-web-access/templates/dispatcher-service.yaml:18]: This must be an internal port only (worker<->dispatcher communication)

File: akeyless-zero-trust-web-access/values.yaml [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+              "Locked": true
+            },
+            "Preferences": {},
+            "WebsiteFilter": {
+              "Block": [
+                "<all_urls>"
+              ],
+              "Exceptions": [
+                "https://*.akeyless.io/*",
+                "https://*.akeyless-security.com/*",
```
Comment (OriMankali) [inline akeyless-zero-trust-web-access/values.yaml:120]: no need for this domain, please remove

**Review by OriMankali — COMMENTED** [review]
Generally looks good, good job.
I still can create bookmarks, this needs to be blocked as it's a security problem, please fix to be as it was in the local deployment.

File: akeyless-zero-trust-web-access/templates/dispatcher-deployment.yaml (hunk 1 above)
Comment (liorhasson) [reply akeyless-zero-trust-web-access/templates/dispatcher-deployment.yaml:53]: this port is open in k8s internal network

File: akeyless-zero-trust-web-access/templates/dispatcher-service.yaml (hunk 2 above)
Comment (liorhasson) [reply akeyless-zero-trust-web-access/templates/dispatcher-service.yaml:18]: good catch! the service (type: load balancer) expose the port to the world, I'm removing it

**Pull request merged**